components/Layout/index.tsx

```diff
@@ -12,48 +18,81 @@ const Wrapper = styled.div`
   width: 100%;
 `
 
-const FullWidthContent = styled.main`
-  margin: -10.4rem auto 0;
-  padding: 0;
-  box-sizing: border-box;
-  width: 100%;
-  display: flex;
-  flex-flow: column wrap;
-`
-
-const FullWidthGradient = styled.main`
-  font-size: 1.6rem;
+const BaseLayout = styled.main`
   display: flex;
   flex-flow: column wrap;
   justify-content: flex-start;
   align-items: flex-start;
   gap: 0;
-  padding: 14rem 0 5.6rem;
   box-sizing: border-box;
   margin: 0 auto;
+  padding: 0;
   width: 100%;
+`
+
+const FullWidthContent = styled(BaseLayout)`
+  margin-top: -10.4rem;
+`
+
+const FullWidthGradient = styled(BaseLayout)`
+  font-size: 1.6rem;
+  padding: 14rem 0 5.6rem;
   min-height: 80rem;
   ${Color.gradientMesh};
   background-size: 100% 100%;
   background-attachment: fixed;
 `
 
+const FullWidthCoWAMM = styled(BaseLayout)`
+  font-size: 1.6rem;
+  padding: 18rem 0 5.6rem;
+  min-height: 80rem;
+  background: ${Color.cowammBlack};
+  color: ${Color.cowammWhite};
+  font-family: ${Font.circular};
+
+  // mobile
+  ${Media.mobile} {
+    padding: 12rem 0 5.6rem;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4 {
+    font-family: ${Font.flecha};
+    font-weight: 500;
+  }
+`
+
 type LayoutProps = PropsWithChildren<{
   fullWidth?: boolean
   fullWidthGradient?: boolean
   fullWidthGradientVariant?: boolean
+  fullWidthCoWAMM?: boolean
 }>
 
-export default function Layout({ children, fullWidth = false, fullWidthGradientVariant = false }: LayoutProps) {
-  const ContentComponent = fullWidth ? FullWidthContent : fullWidthGradientVariant ? FullWidthGradient : Content
-  const FooterNoMargin = fullWidthGradientVariant ? true : false
+export default function Layout({ children, fullWidth, fullWidthGradientVariant, fullWidthCoWAMM }: LayoutProps) {
```

Comment:
nit: same issue with handling all with flags. this variants are mutually exclusive, but handled as flags.